src/01_hello_world_api/README.md

```diff
@@ -6,7 +6,7 @@ In the first stage we're going to create a python web API with [Flask](https://f
 
     >To be able to build the API, we first need to get the package of API we'll be using.
 
-    1. Create a `requirements.txt` file in the `src` folder
+    1. Create a `requirements.txt` file in the `src/` folder
 
         ```sh
```

Comment:
nit: We could add `cd src`
Do you expect the students to create their own app folder in src?

Reply:
I added it :) In the further sections, there are parts where explicit instructions are skipped so that the students must think for themselves.